pages/create-collective.js

```diff
@@ -50,7 +50,7 @@ const CreateCollectivePage = ({ loadingLoggedInUser, LoggedInUser }) => {
   }
 
   return (
-    <Page>
+    <Page showFooter={LoggedInUser}>
```

Comment:
`showFooter` proptype expects a boolean, let's give it what it wants
```suggestion
    <Page showFooter={Boolean(LoggedInUser)}>
```

Reply:
Ah, makes sense. Done. 👍🏽